dm/syncer/data_validator.go

```diff
@@ -78,6 +78,9 @@ const (
 	validatorDmctlOpTimeout = 5 * time.Second
 )
 
+// to make ut easier, we define it as a var, so we can change it.
+var markErrorRowDelay = config.DefaultValidatorRowErrorDelay
```

Comment:
Why is it a global variable rather than a member of the validator (worker)? It cannot be used independently by other components.

Reply:
it should be a const, but as comments says `to make ut easier, we define it as a var, so we can change it.`